code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -27,6 +27,9 @@
 	var/metabolism_min = 0.01 //How much for the medicine to be present in the system to actually have an effect.
 	var/list/conflicting_reagents //Reagents that conflict with this medicine, and cause adverse effects when in the blood.
 
+	var/list/container_whitelist = list()

```

Comment:
Do not make `= list()` vars on commonly instanced types, like reagents.

Reply:
I'll remove the feature entirely.